libs/domain/order/confirmation-banner/confirmation-banner.component.ts

```diff
@@ -10,19 +10,22 @@ export class OrderConfirmationBannerComponent extends OrderMixin(
 ) {
   static styles = styles;
 
-  @asyncState()
-  protected orderRef = valueType(this.orderController.getRef());
+  protected orderRef = signal(this.orderController.getRef());
```

Comment:
What about `orderId`, same in controller

Reply:
I planned to refactor controller and service in another sub task, when everything is on place